### PR TITLE
Fix int64 overflow issue to due precision issues in Typescript with large numbers

### PR DESCRIPTION
**Pull request by ilanashapiro**
Currently, `MAX_INT_64` is set to `2 ** 63 - 1`, and `MIN_INT_64` is set to `0 - 2 ** 63` in `src/oas/transformers/schema/keywords/format.ts`. The TypeScript compiler evaluates these, respectively, to 9223372036854776000 and -9223372036854776000. 

## Motivation and Context
These values are outside the range of int64 and result in bugs in clients such as Prism that use this library (e.g. https://github.com/stoplightio/prism/issues/1992). 

The reasoning behind this erroneous calculation is explained here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER. Essentially, we are overflowing the capacity of JavaScript's Number type, and beyond Number.MAX_SAFE_INTEGER + 1 (9007199254740992), the IEEE-754 floating-point format can no longer represent every consecutive integer (see: https://stackoverflow.com/questions/1379934/large-numbers-erroneously-rounded-in-javascript). 

## Description
Thus, instead of setting `MAX_INT_64` to `2 ** 63 - 1`, and `MIN_INT_64` to `0 - 2 ** 63`, we set them to TypeScript's built-in Number.MAX_SAFE_INTEGER and Number.MIN_SAFE_INTEGER instead. 

## How Has This Been Tested?
Update local tests including screenshot tests.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] This PR's code follows as closely as possible [the coding style/guidelines of this project](???).
- [ ] I have added error reporting and followed [the error reporting guidelines](???).
- [ ] I have added event tracking and followed [the event tracking guidelines](???).
- [ ] I have updated any relevant documentation accordingly to reflect this PR's changes.
- [X] I have added automated tests (unit/integration/e2e/other) to cover my changes.
- [X] All new and existing tests pass locally (excluding flaky CI tests).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
     "lodash.pickby": "^4.6.0",
     "openapi3-ts": "^2.0.2",
     "postman-collection": "^4.1.2",
-    "tslib": "^2.3.1",
+    "tslib": "^2.6.2",
```
Comment (ilanashapiro) [inline package.json:88]: Number.MAX_SAFE_INTEGER and Number.MIN_SAFE_INTEGER are not available on tslib v2.3.1 so we need to update

**Comment by ilanashapiro** [comment]
@daniel-white Hi! Could someone from your team please take a look at this PR? Thanks!

**Comment by P0lip** [comment]
Heyo! I'll take a look at the PR soon

**Review by P0lip — APPROVED** [review]
Ideally, we'd switch these values over to bigints, but that'd be somewhat troublesome to implement and would require changes across many repos and libraries.
I'm fine with this change since at the very least the values are now well within the range.

**Pull request merged**